clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2295,8 +2299,12 @@ def GlobalOp : CIR_Op<"global",
     bool hasAvailableExternallyLinkage() {
       return mlir::cir::isAvailableExternallyLinkage(getLinkage());
     }
+    bool hasInternalLinkage() {
+      return mlir::cir::isInternalLinkage(getLinkage());
+    }
     /// Whether the definition of this global may be replaced at link time.
     bool isWeakForLinker() { return cir::isWeakForLinker(getLinkage()); }
+    bool isDSOLocal() { return getDsolocal(); }
```

Comment:
The difference in capitalization bothers me, but that's not this patch's fault.

Reply:
Yea, that's MLIR ODS. Can't do anything about that here.